libsolidity/lsp/LanguageServer.cpp

```diff
@@ -232,14 +232,15 @@ bool LanguageServer::run()
 {
 	while (m_state != State::ExitRequested && m_state != State::ExitWithoutShutdown && !m_client.closed())
 	{
+		MessageID id;
```

Comment:
Shouldn't this be default-initialized with `nullValue`, or is this the default?

Reply:
It is the default.